src/mon/MonCommands.h

```diff
@@ -1326,6 +1326,15 @@ COMMAND("mgr module enable "
 COMMAND("mgr module disable "
 	"name=module,type=CephString",
 	"disable mgr module", "mgr", "rw")
+COMMAND("mgr module set queue threshold "
+	"name=module,type=CephString "
+	"name=threshold,type=CephInt,range=1|5",
```

Comment:
Do we need to set max range here? If yes, what should be a good value. Currently, it is just a random number

Reply:
It does make sense to allow the cluster operator to tune this limit, but the tricky thing is the default value. Even if its per-plugin (as @batrick is suggesting), do we know what would be a sensible default?

Reply:
Currently, I am setting the default value as `0` which is then being used by the code to check whether the option is set by user or not